crates/rspack_loader_runner/src/runner.rs

```diff
@@ -220,7 +220,9 @@ async fn create_loader_context<'c, C: 'c>(
   context: C,
 ) -> Result<LoaderContext<'c, C>> {
   let mut file_dependencies: HashSet<PathBuf> = Default::default();
-  file_dependencies.insert(resource_data.resource_path.clone());
+  if resource_data.resource_path.is_absolute() {
```

Comment:
If this is only for filter data url resource, use resource_path != "", we can refactor resource_path to Option<String> in the future PR, which aligns with data url resourcePath is undefined in webpack